src/util/make-knex.js

```diff
@@ -54,6 +54,26 @@ function initContext(knexFn) {
       return this.client.ref(ref);
     },
 
+    disableProcessing() {
```

Comment:
Would be good to add some comment that these should not be added documentation to be a part to knex public API before we have designed better API names and parameters for this.

Reply:
And also to mention why these exist (that it was needed for migrations to be able to run those internal queries, with consistent identifier naming).